archai/algos/evolution_pareto/evolution_pareto_search.py

```diff
@@ -1,5 +1,6 @@
 from abc import ABCMeta, abstractmethod
 from overrides.overrides import overrides
```

Comment:
Please add Piero and ask him to review discrete search code. Again, please make sure your changes doesn't break other things. This is super important because we use this in many other projects.